src/randoop/experiments/RandomWalkGenerator.java

```diff
@@ -85,8 +84,7 @@
   private long gentime;
   private long exectime;
 
-  private static SequenceCollection prims = new SequenceCollection(
-      SeedSequences.defaultSeeds());
+  private static SequenceCollection prims = new SequenceCollection(SeedSequences.defaultSeeds());
```

Comment:
Your change is fine and does not need to be modified.
If I were to break the line, I would personally do it before the "=".